tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26,py27,py32,py33,py34,pypy,pypy3,coverage,docs
+envlist = py27,py33,py34,pypy,pypy3,coverage,docs
```

Comment:
b1c86c5  drops that block.  I can't find any actual use of `ordereddict` anywhere in the source.